### PR TITLE
Updated README to add did:web example 

### LINKED ISSUE #322: 🎃 Hacktoberfest: Web5 Rust CLI Enhancements
## 🚀 Goal
Enhance the Web5 Rust CLI by improving documentation, adding new features, and expanding existing functionality.

## 🤔 Background
The Web5 Rust CLI is a powerful tool for working with decentralized identifiers (DIDs) and verifiable credentials (VCs). This project aims to make it more user-friendly and feature-rich.

# 🤝 Pick up  _ONE_ of the following issues to contribute
We've broken down the Web5 Rust CLI implementation into smaller tasks. Feel free to take on any task that interests you. Every contribution counts!

> Each subtask is detailed in its own issue. Please refer to the individual issues for specific requirements and acceptance criteria.

### 💻 Coding Tasks (Rust specific)
- [ ] #326 
- [ ] #327
- [ ] #328 
- [x] #380 
- [ ] #384
- [ ] #386 
- [ ] #387 
- [ ] #388 

### 📚 Documentation
- [x] #323 
- [ ] #324
- [x] #325 

## 📋 Overall Acceptance Criteria
- All subtasks are completed and their respective issues closed
- Documentation is updated to reflect all new features and changes
- New features are thoroughly tested and working as expected
- Code follows the project's style guidelines and best practices
- Pull requests for each subtask are reviewed and approved

## 🌟 Resources
- [Web5 Documentation](https://developer.tbd.website/docs/web5/)
- [Web5 Rust CLI Repository](https://github.com/TBD54566975/web5-rs/tree/main/crates/web5_cli)
- [Web5 CLI built in Go for reference](https://github.com/TBD54566975/web5-go/tree/main/cmd/web5)
- [Rust Documentation Guidelines](https://doc.rust-lang.org/rustdoc/what-is-rustdoc.html)
- [JWT.io](https://jwt.io/) (for JWT-related tasks)

## 🎊 Hacktoberfest
This project is part of Hacktoberfest! We welcome contributions from technologists of all skill levels. Here's how you can get started:

1. Read the [contributing guidelines](#)
2. Read the [code of conduct](#)
3. Choose a subtask you'd like to work on
4. Comment on the subtask's issue to get assigned
5. Fork the repository and create a new branch for your work
6. Complete the subtask following the criteria in its issue
7. Submit a pull request with your changes
8. Respond to any feedback during the review process

## Questions?
If you have any questions or need clarification on any of the subtasks, please comment on the relevant issue or join our [Discord](https://discord.gg/tbd) community!

Happy coding! 🎉

### PR DESCRIPTION
**Pull request by aazam-gh**
Fixes #322

### CODE REVIEW

File: crates/web5_cli/README.md [hunk 1]
```diff
@@ -62,6 +62,91 @@ web5 vc -h
 }
 ```
 
+### Create a `did:jwk`
```
Comment (KendallWeihe) [inline crates/web5_cli/README.md:65]: @aazam-gh could you rebase onto main? (The diff viewer is showing changes from #379)

**Comment by blackgirlbytes** [comment]
@aazam-gh let me know if you need help with the rebase. If it makes it easier, feel free to close this PR, pull the latest changes from main, make a new branch and then open a new PR

**Comment by aazam-gh** [comment]
Is the diff still showing the changes?

**Review by KendallWeihe — APPROVED** [review]
thank you @aazam-gh !

**Comment by blackgirlbytes** [comment]
> Is the diff still showing the changes?

it looks like it's still showing the changes from the past PR to me.

**Comment by blackgirlbytes** [comment]
> Closes #324

**Pull request merged**